tools/rosgraph/CMakeLists.txt

```diff
@@ -17,4 +17,5 @@ catkin_install_python(PROGRAMS scripts/rosgraph
 
 if(CATKIN_ENABLE_TESTING)
   catkin_add_nosetests(test)
+  catkin_add_nosetests(test/isolated/test_roslogging_user_logger.py)
```

Comment:
Actually, the test fails on my PC if I run nosetests for test directory.
below shows the cases. The cause seems the logger at running time, but not sure how to fix this at this time.

```
$ ls test
test_ros_logging_user_logger.py  ....py ....py  ....py
$ nosetests test  # fails
$ nosetests test/test_ros_logging_user_logger.py  # success
```